app/src/main/java/com/example/kuba10/mypokemonplaces/ListFragment/FirebaseListFragment.java

```diff
@@ -1,151 +0,0 @@
-package com.example.kuba10.mypokemonplaces.ListFragment;
```

Comment:
Z tą klasą będzie pewnie najwięcej problemów. Generalnie sugeruję, że skoro już jesteś w miejscu w którym nie masz żadnych pomysłów, to teraz wykonać refactor. W dużej mierze poprawi czytelność kodu oraz, co bardzo możliwe, pozwoli na zidentyfikowanie problemów.